### PR TITLE
Publish to hex

### PR DESCRIPTION
**Pull request by myronmarston**
* https://hex.pm/packages/public_suffix
* https://hexdocs.pm/public_suffix/api-reference.html

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /deps
 erl_crash.dump
 *.ez
+doc
```
Comment (GreyKn) [inline .gitignore]: `.DS_Store` is generally a good file to ignore by default as well

Comment (myronmarston) [inline .gitignore]: Done.

File: mix.exs [hunk 2]
```diff
@@ -24,20 +24,42 @@ defmodule PublicSuffix.Mixfile do
...
+      {:ex_doc, ">= 0.0.0", only: :dev},
+      {:earmark, ">= 0.0.0", only: :dev},
     ]
   end
+
+  defp description do
+    """
+    Operate on domain names using the public suffix rules provided by https://publicsuffix.org/.
+    """
+  end
```
Comment (benkirzhner) [inline mix.exs]: Thoughts about making this a module attribute instead of a private method?

Comment (myronmarston) [inline mix.exs]: The mix convention is to define it as a private function.  I got this directly from the publishing guide:

https://hex.pm/docs/publish

Consider also that in `mix.exs` you usually define `defp deps`, `defp application`, etc, when those things could be module attributes.

Comment (benkirzhner) [inline mix.exs]: I hadn't seen those docs before; makes sense.

**Comment by benkirzhner** [comment]
LGTM

**Pull request merged**